net.krafting.PedantiK.yml

```diff
@@ -0,0 +1,66 @@
+app-id: net.krafting.PedantiK
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: PedantiK.py
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: OpenBLAS
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    sources:
+      - type: git
+        url: https://github.com/OpenMathLib/OpenBLAS
+        tag: 'v0.3.28'
+        commit: 5ef8b1964658f9cb6a6324a06f6a1a022609b0c5
+
+  - name: meson-python
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/mesonbuild/meson-python
+        tag: '0.15.0'
+        commit: e847bdd918ba6686d9b0502e9ee6d83d5145ae84
```

Comment:
this should be cleaned up with `- "*"` as cleanup

Reply:
should be good